plugins/armorblox/unit_test/payloads/get_remediation_action.json

```diff
@@ -0,0 +1,49 @@
+{
+   "priority":"HIGH",
+   "tagged":false,
+   "date":"2022-08-16T06:51:51Z",
+   "users":[
+      {
+         "name":"James Pardue",
+         "email":"james@armorblox.dev",
```

Comment:
Please replace these with fake names and emails - we have done this in other PRs like Cortex

Reply:
Sure Rajat!